### DISCUSSION TITLE
Add example for tweens animation

### DISCUSSION THREAD

**Discussion by Bouh** [4ian/GDevelop#2333] [Enhancement ideas (and their technical discussions)]
Hi @Wend1go I remember when you have added Tween animations you have made a example for test each types of tweening.

This example is very helpful i think it should be include with the other examples.

## Solution suggested
Use this file somewhere in a **test** folder in the GDevelop repo and move it in **example** folder.
https://github.com/4ian/GDevelop/pull/922#issuecomment-467959740

**Comment by 4ian** [comment]
I do think that would be a good example. To add one, creating a folder in the newIDE/app/resources/examples with a README.md is enough. Then you have to run `node update-fixtures-from-resources-examples.js` and `node update-examples-information-from-resources-examples.js` in the newIDE/app/scripts folder. Finally for the web-app I'll upload myself the necessary resources.

**Comment by TheGemDev** [comment]
I think I should do this after the solarised dark theme

**Comment by 4ian** [comment]
This was done :) Locking the discussion.
